giraffe/src/transforms/band.ts

```diff
@@ -437,38 +438,28 @@ export const bandTransform = (
       lineData[groupID] = {xs: [], ys: [], fill: ''}
       // 'fill' is set temporarily to no color
       //    it will be updated with another loop later,
-      //    because it is faster to walk bandIndexMap once
+      //    because it is faster to walk bandLineMap once
       //    than to search for the indices of many lines
     }
 
     lineData[groupID].xs.push(x)
     lineData[groupID].ys.push(y)
 
-    // remember the latest (most recent) index for each group
```

Comment:
Sorry, asking bc i just dont have context into this, so im curious - what is the purpose of this deleted code and why is it being removed?

Reply:
That chunk of code was causing the misalignment and inconsistencies between the tooltip and the static legend. It is being replaced further down in the file by a function that will be shared between tooltip and static legend.